lib/public/views/Runs/format/displayEnvironmentLinks.js

```diff
@@ -0,0 +1,32 @@
+/**
+ * @license
+ * Copyright CERN and copyright holders of ALICE O2. This software is
+ * distributed under the terms of the GNU General Public License v3 (GPL
+ * Version 3), copied verbatim in the file "COPYING".
+ *
+ * See http://alice-o2.web.cern.ch/license for full licensing information.
+ *
+ * In applying this license CERN does not waive the privileges and immunities
+ * granted to it by virtue of its status as an Intergovernmental Organization
+ * or submit itself to any jurisdiction.
+ */
+
+import { absoluteFrontLink } from '../../../components/common/navigation/absoluteFrontLink.js';
+import { frontLink } from '../../../components/common/navigation/frontLink.js';
+import { isRunRunning } from '../../../utilities/formatting/isRunRunning.js';
+const { CONFIGURATION: { ALI_ECS_GUI_URL } } = window;
+import { h } from '/js/src/index.js';
+
+/**
+ * Format environment links
+ * @param {object} run the run
+ * @return {Componenet} links
+ */
+export const displayEnvironmentLinks = ({ environmentId: id, ...run }) => isRunRunning(run)
```

Comment:
I think that here you are going to complex: You should simply check that :

`timeTrgEnd ?? timeO2End ?? null === null && runDuration < MAX_RUN_DURATION`
You don't need the difference between run lost vs lost actually stopped.

You can create a function, like `isRunConsideredRunning` but in this file, because you don't reuse it elsewhere.

Reply:
applied